material/problems/VectorGeometry/PolygonNormal/PolygonNormal.tex

```diff
@@ -1,65 +1,125 @@
+\plus[\;] 0~\uvecz{a}$.
+%
+\begin{center}
+  \includegraphicsAB[width=7cm]{polygon-solution.png}{polygon_normal.png}
+\end{center}
+%
 \begin{enumerate}
-\item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon, expressed in \uvecxyz{a}.
-\item \underline{Explain} in detail how to get \uvecWithHat{n}, the unit vector in the same direction as \bvec{n}. \underline{Sketch} \uvecWithHat{n} on the figure.
-\item \underline{Explain} why there are actually two possible \uvecWithHat{n} vectors for the polygon.
-%\item Compute \posvec{P_0}{Q}, and hence compute the unit vector $\uvecWithHat{L}$.
-%\item For a light source of strength $I_L = 0.5$, compute the diffuse illumination $I_D$.
+  \item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon,
+  expressed in terms of \uvecxyz{a}.
+    \\[0.25pc]\textbf{Result:} \\[-1.0pc]
+    %
+    \begin{displaymath}
+      \bvec{n} \equals[\;]
+      \hidemath[2cm]{4~\uvecx{a} \plus[\;] 6~\uvecy{a} \plus[\;] 8~\uvecz{a}}
+    \end{displaymath}
+    %
+    \Solution{}{0.99\linewidth}{
+        \begin{tabular}{p{0.03\linewidth}ll}
+          $\bvec{n}$
+          & $\equals[\;] \posvec{P_2}{P_1} \CrossProduct[\;] \posvec{P_3}{P_1}$
+          \\[0.45pc]
+          %\begin{align*}
+          $\posvec{P_2}{P_1}$
+          & $\equals[\;] \posvec{A_o}{P_2} \minus[\;] \posvec{A_o}{P_1}$
+          & $\equals[\;] (2 - 3)~\uvecx{a} \plus[\;] (0 - 2)~\uvecy{a}
+          \plus[\;] (2 - 0)~\uvecz{a}$
+          \\[0.45pc]
+          & & $\equals[\;] -1~\uvecx{a} \minus 2~\uvecy{a} \plus[\;] 2~\uvecz{a}$
+          %\end{align*}
+          \\[0.45pc]
+          $\posvec{P_3}{P_1}$
+          & $\equals[\;] \posvec{A_o}{P_3} \minus[\;] \posvec{A_o}{P_1}$
+          & $\equals[\;] (6 - 3)~\uvecx{a} \plus[\;] (0 - 2)~\uvecy{a}
+          \plus[\;] (0 - 0)~\uvecz{a}$
+          \\[0.45pc]
+          & & $\equals[\;] 3~\uvecx{a} \minus 2~\uvecy{a}$
+          \\[0.45pc]
+          $\bvec{n}$
+          & $\equals[\;] \posvec{P_2}{P_1} \CrossProduct[\;] \posvec{P_3}{P_1}$
+          & $\equals[\;] (-1~\uvecx{a} \minus 2~\uvecy{a} \plus[\;] 2~\uvecz{a})
+          \CrossProduct[\;] (3~\uvecx{a} \minus 2~\uvecy{a})$
+          \\[0.45pc]
+          & & $\equals[\;] -3~\uvecx{a} \cdot ~\uvecx{a} \plus[\;]
+          2~\uvecy{a} \cdot ~\uvecx{a} \minus[\;] 6~\uvecy{a}
+          \cdot ~\uvecx{a} $
+          \\[0.45pc]
+          & & \quad\; $\plus[\;] 4~\uvecy{a} \cdot ~\uvecy{a} \plus[\;]
+          6~\uvecz{a} \cdot ~\uvecx{a} \minus[\;] 4~\uvecz{a} \cdot ~\uvecy{a}$
+          \\[0.45pc]
+          & & $\equals[\;] 2~\uvecz{a} \minus 6(-~\uvecx{a}) \plus[\;] 6~\uvecy{a}
+          \minus 4(-~\uvecx{a})$
+          \\[0.45pc]
+          & & $\equals[\;] 4~\uvecx{a} \plus[\;] 6~\uvecy{a} \plus[\;] 8~\uvecz{a}$
+        \end{tabular}
+    }
+  \item \underline{Explain} in detail how to get \uvecWithHat{n}, the unit vector
```

Comment:
There's a shorter version of this, called `\uvecHat`. Let's use that everywhere instead.